src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is linking to a SCSS file (`styles/main.scss`). Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Ensure that you link to the compiled CSS file instead.